### PR TITLE
Update bichard7 repositories

### PR DESCRIPTION
**Pull request by bjpirt**
- Add a new repo
- Remove a user from the existing repos

### CODE REVIEW

File: terraform/bichard7-next-pnc-emulator.tf [hunk 1]
```diff
@@ -0,0 +1,66 @@
+module "bichard7-next" {
```
Comment (jakemulley) [inline terraform/bichard7-next-pnc-emulator.tf:1]: @bjpirt This key needs to be updated (it's already in use).

**Review by jakemulley — CHANGES_REQUESTED** [review]

File: terraform/bichard7-next-pnc-emulator.tf (hunk 1 above)
Comment (bjpirt) [reply terraform/bichard7-next-pnc-emulator.tf]: Thanks - good old copy-paste!

**Comment by bjpirt** [comment]
@jakemulley I'm conscious that we have a number of repos with the same contributors - is there a way we could dry this up that would work with this repo structure?

**Comment by jakemulley** [comment]
@bjpirt You could use `locals` to define a local variable, if it's all the same people in each of the repositories, and use it as `collaborators = local.bichard7-collaborators` to remove the duplication 👍🏼

**Comment by AntonyBishop** [comment]
@jakemulley are you ok with the amendments?

@bjpirt any further commits you want to make?

**Review by jakemulley — APPROVED** [review]

**Comment by jakemulley** [comment]
@AntonyBishop 👍🏼 The `locals` change, if @bjpirt wants to implement it, is OK to go in as another PR 👍🏼

**Comment by AntonyBishop** [comment]
Cheers @jakemulley

**Pull request merged**